src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/profiles.yml

```
@@ -0,0 +1,38 @@
+admin_profiles:
```

Check naming convention :+1: 
```suggestion
admin_profiles_index:
```